drivers/video/video_esp32_dvp.c

```diff
@@ -23,6 +24,16 @@
 LOG_MODULE_REGISTER(video_esp32_lcd_cam, LOG_LEVEL_INF);
 
 #define VIDEO_ESP32_DMA_BUFFER_MAX_SIZE 4095
+#define VIDEO_ESP32_VSYNC_MASK 0x04
+
+#ifdef CONFIG_POLL
+#define VIDEO_ESP32_RAISE_OUT_SIG_IF_ENABLED(result) \
+	if (data->signal_out) { \
+		k_poll_signal_raise(data->signal_out, result); \
+	}
```

Comment:
It could be a good idea to provide this macro in `video.h` but this can be done in a separate PR.